From 4d901187c6bc2988023889d33ea45ab30cb5da6d Mon Sep 17 00:00:00 2001
From: Eric Courtois <eric.courtois@gmail.com>
Date: Sat, 20 Nov 2021 20:20:03 +0100
Subject: [PATCH 3/4] protoc plugin: comment out deprecated marks on functions

RIOT is compiled using -Werror, so functions marked as deprecated generate
compilation errors.
So remove the deprecated marks.

Signed-off-by: Eric Courtois <eric.courtois@gmail.com>
---
 src/ReadBufferInterface.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ReadBufferInterface.h b/src/ReadBufferInterface.h
index d3f8638..a0386ed 100644
--- a/src/ReadBufferInterface.h
+++ b/src/ReadBufferInterface.h
@@ -65,14 +65,14 @@ namespace EmbeddedProto
       virtual bool peek(uint8_t& byte) const = 0;
 
       //! Advances the internal read index by one when the buffer is not empty.
-      [[deprecated("From version 3.0.0 onwards this function will return a boolean.")]] virtual void advance() = 0;
+      /*[[deprecated("From version 3.0.0 onwards this function will return a boolean.")]]*/ virtual void advance() = 0;
 
       //! Advances the internal read index by the given value.
       /*!
           The advance is limited to the number of bytes in the buffer.
           \param[in] n_bytes The number of bytes to advance the read index.
       */
-      [[deprecated("From version 3.0.0 onwards this function will return a boolean.")]] virtual void advance(const uint32_t n_bytes) = 0;
+      /*[[deprecated("From version 3.0.0 onwards this function will return a boolean.")]]*/ virtual void advance(const uint32_t n_bytes) = 0;
 
       //! Obtain the value of the oldest byte in the buffer and remove it from the buffer.
       /*!
-- 
2.25.1

